src/main/java/games/strategy/engine/framework/ArgParser.java

```diff
@@ -120,10 +120,42 @@ private boolean setSystemPropertyOrClientSetting(
   }
 
   private static boolean handleGameSetting(final String key, final String value) {
-    if (GameRunner.MAP_FOLDER.equals(key)) {
+    if (CliProperties.MAP_FOLDER.equals(key)) {
       ClientSetting.MAP_FOLDER_OVERRIDE.save(value);
       return true;
     }
     return false;
   }
+
+
+  /**
+   * An interface for all CLI related constants.
+   */
+  public interface CliProperties {
```

Comment:
This is definitely a step in the right direction.  The only thing I'm not sure about is if `ArgParser` is the correct place to host these constants.  But since I don't have any good suggestions right now, I'd say let's leave things as they are in this PR. :smile:  As we dig into this more in future PRs and try to tease apart the dependency on system properties, we should "discover" the correct code structure.